test/test.js

```diff
@@ -193,11 +193,11 @@ describe('mockExpress', function() {
 		done();
 	});
 
-	it('should return status on res.status', function(done) {
+	it('should return status on res.sendStatus', function(done) {
```

Comment:
We should not remove the test for `status`, but test both separately. In addition, `res.sendStatus` should also be tested to return strings as [described in the API](http://expressjs.com/en/api.html#res.sendStatus).